<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="listGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>701</height>
    </rect>
   </property>
   <property name="title">
    <string>Controller List</string>
   </property>
   <widget class="QPushButton" name="btnCCTV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CCTV 관제실</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnControll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Conveyor Controll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMonitoring">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Conveyor Monitoring</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableKedex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kedex Viewer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cctvControll">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>581</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>CCTV 보관실</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="cctvLayout"/>
   </widget>
   <widget class="QComboBox" name="cctvOrderComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="conveyorMonitoring">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>641</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Conveyor Monitoring</string>
   </property>
   <widget class="QLabel" name="barcodeLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>barcodeLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConveyorSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelConveyorTime">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelConveoyStatus">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="conveyorStatusTitle">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelConveyorDate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="conveyorStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>conveyorStartButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conveyorStopButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>conveyorStopButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConveyorCamera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="SliderConveyorSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>621</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="receiveData">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>receiveData</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="conveyorControll">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>661</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Conveyor Statistics</string>
   </property>
   <widget class="QLabel" name="chartLabel_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>chartLabel_3</string>
    </property>
   </widget>
   <widget class="QLabel" name="chartLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>chartLabel_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="chartLabel_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>chartLabel_1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
